IBAnimatable/AnimatableViewController.swift

```diff
@@ -30,9 +30,10 @@ import UIKit
   public override func viewWillDisappear(_ animated: Bool) {
     super.viewWillDisappear(animated)
     resetHideNavigationBar()
+
   }
   
-  public override func preferredStatusBarStyle() -> UIStatusBarStyle {
+  public override var  preferredStatusBarStyle:UIStatusBarStyle{
```

Comment:
there is a small spacing issue. `public override var preferredStatusBarStyle: UIStatusBarStyle {`